src/services/suggestionDiagnostics.ts

```diff
@@ -213,4 +213,16 @@ namespace ts {
 
         return false;
     }
+
+    export function isAsyncLikeFunction(node: Node): node is FunctionDeclaration | MethodDeclaration | FunctionExpression | ArrowFunction {

```

Comment:
I would potentially rename this to `canBeConvertedToAsync`

```suggestion
    export function isFunctionThatSupportsAsync(node: Node): node is FunctionDeclaration | MethodDeclaration | FunctionExpression | ArrowFunction {
```

Reply:
Renamed to `canBeConvertedToAsync`